GitCommands/Remotes/ConfigFileRemoteSettingsManager.cs

```diff
@@ -164,12 +165,19 @@ bool IsSettingForBranch(string setting, string branchName)
         /// </summary>
         public IReadOnlyList<Remote> GetDisabledRemotes()
         {
-            var disabledRemotes = GetModule().LocalConfigFile.GetConfigSections()
-                .Where(s => s.SectionName == $"{DisabledSectionPrefix}remote")
-                .Select(s => new Remote(s.SubSection, s.GetValue("url"), s.GetValue("pushurl", s.GetValue("url"))))
-                .ToList();
+            return GetDisabledRemotes().ToList();
 
-            return disabledRemotes;
+            IEnumerable<Remote> GetDisabledRemotes()
+            {
+                foreach (IConfigSection section in GetModule().LocalConfigFile.GetConfigSections())
+                {
+                    if (section.SectionName == $"{DisabledSectionPrefix}remote")
+                    {
+                        Validates.NotNull(section.SubSection);
+                        yield return new Remote(section.SubSection, section.GetValue("url"), section.GetValue("pushurl", section.GetValue("url")));
+                    }
+                }
+            }
```

Comment:
Rewrote this to allow validation of `SubSection` as non-null. Roslyn should probably detect this automatically, based upon it being `==` to a non-null value. However `==` is overridable, so it'd have to special-case `string` which seems narrow in scope.